controls/5_4_user_accounts_and_environments.rb

```diff
@@ -176,7 +176,7 @@
   tag level: 1
 
   describe passwd.users('root') do
-    its('gids') { should cmp 0 }
```

Comment:
![Screen Shot 2022-11-03 at 3 18 05 PM](https://user-images.githubusercontent.com/109251274/199836102-6811c05e-c701-4558-8120-edc9e7fd3c4d.png)

Reply:
`be_grouped_into` should be `be_owned_by` instead.

```
    it { should be_owned_by 'root' }
```
However I think in this check is explicitly checking if the group account number is 0. So I'm going to revert this change to how it was